src/unix/linux-core.c

```diff
@@ -829,19 +829,21 @@ static int uv__ifaddr_exclude(struct ifaddrs *ent, int exclude_type) {
 int uv_interface_addresses(uv_interface_address_t** addresses,
   int* count) {
```

Comment:
nit: while you're here, could rewrite the function declaration in one line only?